doc/tutorials/figures/.directory

```diff
@@ -0,0 +1,4 @@
+[Dolphin]
```

Comment:
what is this stuff for?

Reply:
Oops, this looks like a file added by my file browser. I'll delete it.